opentelemetry-otlp/src/exporter/mod.rs

```diff
@@ -76,19 +77,19 @@ pub struct ExportConfig {
     pub protocol: Protocol,
 
     /// The timeout to the collector.
-    pub timeout: Duration,
+    pub timeout: Option<Duration>,
```

Comment:
strange that we exposed this struct without a builder!
Can you add doc comments to indicate configuring via this would be final, and cannot be overridden by env?